.github/workflows/autobuild.yml

```diff
@@ -3,123 +3,132 @@
+             building_on_os:        macos-10.15
              asset_path:            deploy/Jamulus-installer-mac.dmg
-             asset_name:            jamulus_${{ needs.create_release.outputs.version }}_mac.dmg
-             asset_latest_name:     jamulus_latest_mac.dmg
-           #- os:                   windows-latest
+             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_mac.dmg
+           #- config_name:          Windows
+           #  target_os:            windows
+           #  building_on_os:       windows-latest
            #  asset_path:           ${{ github.workspace }}\deploy\Jamulus-installer-win.exe
            #  asset_latest_name:    jamulus_latest_win.exe
-           #  asset_name:           jamulus_${{ needs.create_release.outputs.upload_url }}_win.exe
-
+           #  asset_name:           jamulus_${{ needs.create_release.outputs.version_name }}_win.exe
+           - config_name:           AndroidAPK
+             target_os:             android
+             building_on_os:        ubuntu-18.04
+             asset_path:            android-build/build/outputs/apk/debug/android-build-debug.apk
+             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_android.apk
     steps:
-
-      ### Setup ###
-
-      # Get the code
+      # Checkout code
       - name:                       Checkout code
         uses:                       actions/checkout@v2
-      # Install Qt (currently Windows only)
+        with:
+          submodules:               true
+        
+      # Install QT for windows (part of scripts in other os)
       - name:                       Install Qt (64 Bit)
         uses:                       jurplel/install-qt-action@v2
-        if:                         matrix.config.os == 'windows-latest'
+        if:                         matrix.config.building_on_os == 'windows-latest'
         with:
           version:                  '5.15.2'
           dir:                      '${{ github.workspace }}'
           arch:                     'win64_msvc2019_64'
       - name:                       Install Qt (32 Bit)
         uses:                       jurplel/install-qt-action@v2
-        if:                         matrix.config.os == 'windows-latest'
+        if:                         matrix.config.building_on_os == 'windows-latest'
         with:
           version:                  '5.15.2'
           dir:                      '${{ github.workspace }}'
           arch:                     'win32_msvc2019'
-
-      ### Build ###
-
+        
+      # Build
       - name:                       "Build deb (Linux)"
         run:                        sh linux/autorelease_linux.sh ${{ github.workspace }}
-        if:                         matrix.config.os == 'ubuntu-18.04'
+        if:                         matrix.config.target_os == 'linux'
       - name:                       "Build (Windows)"
```

Comment:
Should be removed and synced to my changes.

Reply:
how? please make a suggestion

Reply:
If you've removed the Install Qt steps (I think this is now the case) and run my script autobuild_windows.ps1 this is solved. I think this PR is no longer up to date.